package.json

```diff
@@ -23,25 +23,39 @@
   "license": "Apache-2.0",
   "homepage": "https://github.com/cerner/terra-aggregate-translations#readme",
   "bugs": {
-    "url": "https://github.com/cerner/terra-aggregate-translations/issues"
+    "url": "https://github.com/cerner/terra-script/issues"
```

Comment:
Let's leave this as `terra-aggregate-translations`

Reply:
whoops